operator/pkg/controllers/suite_test.go

```diff
@@ -54,9 +55,10 @@ var _ = BeforeSuite(func() {
 	logf.SetLogger(zap.New(zap.WriteTo(GinkgoWriter), zap.UseDevMode(true)))
 
 	By("bootstrapping test environment")
+
 	testEnv = &envtest.Environment{
 		CRDDirectoryPaths:     []string{filepath.Join("..", "..", "config", "crd", "bases")},
-		BinaryAssetsDirectory: filepath.Join("..", "..", "testbin", "bin"),
+		BinaryAssetsDirectory: os.Getenv("KUBEBUILDER_ASSETS"),
```

Comment:
Which is the place to set this env? Do we need to set it by using `export KUBEBUILDER_ASSETS=...` manually every time?

Reply:
Originally, it is set by `setup-envtest.sh` script, and the new tool `setup-envtest` also set that by this command in makefile (this change is here: https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.9.0)

```
source <(setup-envtest use -i -p env 1.21.x)
```

for more information for new tool, please take a look https://github.com/kubernetes-sigs/controller-runtime/tree/master/tools/setup-envtest

Reply:
Understood, thx